### ISSUE TITLE
TYPO3 13.4 compatibility

### ISSUE THREAD

**Issue by weblution** [dmitryd/dd_deepl#48]
Hi there, are there plans to make this extension compatible for TYPO3 13.4?

**Comment by kauz56** [comment]
@weblution Our fork is v13 compatible now, if you want to check it out: https://github.com/kauz56/dd_deepl
Treat this as "alpha" software though, testing has been limited so far. If you encounter any bugs that you believe are caused by one of our changes feel free to contact me!

**Comment by dmitryd** [comment]
Yes, it should be ready soon. I did not have much time before... :(

**Comment by dmitryd** [comment]
Version for v13 is out now.
